src/components/App.tsx

```diff
@@ -128,8 +137,6 @@ export default class App extends Component<AppProps, AppState> {
         console.log(configObject);
         let output = "";
 
-        const submitButton = document.getElementById("submit-button") as HTMLInputElement;
-        submitButton.disabled = true;
```

Comment:
I'm excited to see how the arrow button looks, but I don't want to remove this functionality

Reply:
I wanted to avoid the duplication of having the image everywhere, but I can copy it back in.

What's the functionality you're worried about here? Submit with enter? Other accessibility issues?

Reply:
So since we only have two buttons for now, I guess I'll just remove the component and put everything in the main app component?

Reply:
sorry, I meant the functionality of disabling the submit button while the input is being processed

Reply:
Ooh, yeah, gotcha. Yeah, I'll figure that out.